addons/mail/static/src/components/channel_member/channel_member.js

```diff
@@ -13,6 +13,13 @@ export class ChannelMember extends Component {
         return this.props.record;
     }
 
+    /**
+     * @private
+     * @returns {Boolean}
+     */
+    get _hasOpenChat() {
+        return this.channelMemberView.channelMember.partner.id;
+    }
```

Comment:
This should be a method on channelMemberView model, and no need to make it private.
Also checking for id is not necessary, existence of partner is enough.

Reply:
Here we will have to check the `id` because guest members will also have an instance of `Partner` but with no id.

Reply:
This is a bug that needs to be fixed in python actually (another task), guest shouldn't have partner.